lib/rom/sql/attribute.rb

```diff
@@ -235,6 +257,11 @@ def sql_expr
         @sql_expr ||= (meta[:sql_expr] || Sequel[to_sym])
       end
 
+      # @api private
+      def type_extensions
+        @type_extensions ||= TypeExtensions[unwrap_type(type)] || EMPTY_HASH
+      end
```

Comment:
Can't we just pass type extensions when schema is building its attributes? This would eliminate the need for `unwrap_type` run-time method as well as `type_extensions` with lazy-assigment.